src/npc.h

```diff
@@ -264,6 +264,25 @@ struct npc_combat_rules : public JsonSerializer, public JsonDeserializer
     void deserialize(JsonIn &jsin) override;
 };
 
+struct npc_misc_rules : public JsonSerializer, public JsonDeserializer
+{
+    bool allow_pick_up;
+    bool allow_bash;
+    bool allow_sleep;
+
+    npc_misc_rules()
+    {
+        allow_pick_up = true;
+        allow_bash = true;
+        allow_sleep = false;
+    };
+
+    using JsonSerializer::serialize;
+    void serialize(JsonOut &jsout) const override;
+    using JsonDeserializer::deserialize;
+    void deserialize(JsonIn &jsin) override;
+};
```

Comment:
Why a new separate structure for those rules, why not just extend the existing `ncp_combat_rules`? It could be renamed to `npc_ai_rules` or `npc_follower_rules` or similar. I see no use for two separate structures here, when are they aver used separately.